doc/reference/common-weaknesses/execution-cost.rst

```diff
@@ -0,0 +1,18 @@
+Execution cost limits may prevent recovery of funds
```

Comment:
My intention was for this to be covered by https://plutus.readthedocs.io/en/latest/reference/common-weaknesses/hard-limits.html . We could include this as a more specific instance of that one.